content/en/agent/guide/heroku-ruby.md

```diff
@@ -164,41 +164,33 @@ mkdir -p datadog/conf.d/
 Create a configuration file called `postgres.yaml` replacing with your host, dbname, username, and password with the information you got in the previous command:
 
 ```yaml
-   init_config:
-
-   instances:
-     ## @param host - string - required
-     ## The hostname to connect to.
-     ## NOTE: Even if the server name is "localhost", the agent connects to
-     ## PostgreSQL using TCP/IP, unless you also provide a value for the sock key.
-     #
-     - host: "<HOST>"
-
-       ## @param port - integer - required
-       ## Port to use when connecting to PostgreSQL.
-       #
-       port: 5432
-
-       ## @param user - string - required
-       ## Datadog Username created to connect to PostgreSQL.
-       #
-       username: <USERNAME>
-
-       ## @param pass - string - required
-       ## Password associated with the Datadog user.
-       #
-       password: "<PASSWORD>"
-
-       ## @param dbname - string - optional - default: postgres
-       ## Name of the PostgresSQL database to monitor.
-       ## Note: If omitted, the default system postgres database is queried.
-       #
-       dbname: "<DB_NAME>"
-
-       ## @param ssl - string - optional - default: false
-       ## This option determines whether or not and with what priority a secure SSL TCP/IP connection
-       ## is negotiated with the server. There are six modes:
-       ssl: 'true'
+init_config:
+
+instances:
+  - host: <YOUR HOSTNAME>
+    port: <YOUR PORT>
+    username: <YOUR USERNAME>
+    password: <YOUR PASSWORD>
+    dbname: <YOUR DBNAME>
+    ssl: True
+```
+
+Instead of manually replacing and harcoding the configuration, you can also set up your Postgres integration based on Heroku environment variables, using the [prerun script][14] to replace those values before starting the Datadog agent:
```

Comment:
```suggestion
Instead of manually updating the configuration, you can set up your Postgres integration based on Heroku environment variables using the [prerun script][14] to replace those values before starting the Datadog Agent:
```